tests/step-117.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/sh
+# Test:
+#   Test local urls of shared hooks to not be cloned and updated
+
+if ! sh /var/lib/githooks/install.sh; then
+    echo "! Failed to execute the install script"
+    exit 1
+fi
+
+# Make repo
+mkdir -p /tmp/test117 && cd /tmp/test117 || exit 1
+git init &&
+    mkdir ".githooks" &&
+    touch ".githooks/trust-all" &&
+    git add . &&
+    git commit -a -m 'Initial commit' ||
+    exit 1
+
+# Simulate a shared hook repo on the server
+mkdir -p /tmp/shared/shared-server.git &&
+    cd /tmp/shared/shared-server.git &&
+    git init --bare &&
+    git clone /tmp/shared/shared-server.git /tmp/shared/shared.git &&
+    cd /tmp/shared/shared.git &&
+    mkdir -p .githooks/pre-commit &&
+    git config core.hooksPath "/dev/null" && # dont execute hooks in this repo!
+    git checkout -b testbranch &&
+    echo 'echo "Shared hook: test1"' >.githooks/pre-commit/sample-one &&
+    git add . &&
+    git commit -a -m 'Testing' &&
+    git commit --allow-empty -m 'Testing...' &&
+    git checkout -b testbranch2 &&
+    echo 'echo "Shared hook: test2"' >.githooks/pre-commit/sample-one &&
+    git commit -a -m 'Testing 2' &&
+    git push -u origin testbranch testbranch2 ||
+    exit 1
+
+# Make the shared hook repo (clone, reset to a previous commit)
+git clone /tmp/shared/shared.git --branch testbranch /tmp/shared/shared-clone.git &&
+    cd /tmp/shared/shared-clone.git &&
+    git reset --hard HEAD~ &&
+    MD5SUM=$(find /tmp/shared/shared-clone.git -type f -exec md5sum {} \; | md5sum) ||
+    exit 1
+
+cd /tmp/test117 || exit 1
+OUT=$(git hooks shared add --shared /tmp/shared/shared-clone.git 2>&1)
+# shellcheck disable=SC2181
+if ! echo "$OUT" | grep -q "to the local shared hooks is forbidden"; then
+    echo "! Expected adding local path to local shared hooks to fail: $OUT" >&2
+    exit 1
+fi
+
+OUT=$(git hooks shared add --shared file:///tmp/shared/shared-clone.git 2>&1)
+# shellcheck disable=SC2181
+if ! echo "$OUT" | grep -q "to the local shared hooks is forbidden"; then
+    echo "! Expected adding local url to local shared hooks to fail: $OUT" >&2
+    exit 1
+fi
+
+echo "file://///tmp/shared/shared-cloned.git" >.githooks/.shared || exit 1
```

Comment:
A bit too many `/` here?
```suggestion
echo "file:///tmp/shared/shared-cloned.git" >.githooks/.shared || exit 1
```

Reply:
Thats for a reason :-) just to test :-)